packages/dashboards/src/lib/components/widget/widget-header/widget-header.component.html

```diff
@@ -27,8 +27,10 @@
 
                 <!-- link version of the header -->
                 <div *ngIf="url && !editMode">
-                    <a [href]="url" target="_blank" (mousedown)="$event.stopPropagation(); $event.preventDefault()"
-                       class="nui-text-widget nui-widget__header__content-title"
+                    <a [href]="url" (click)="changeLink($event)" target="_blank"
```

Comment:
do we need the `changeLink` call on both mouseenter and click? maybe mouseenter is enough?